code/modules/goonchat/browserassets/html/browserOutput.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Chat</title>
+	<meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
+	<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+	<link rel="stylesheet" type="text/css" href="font-awesome.css" />
+	<link rel="stylesheet" type="text/css" href="browserOutput.css" />
+	<script type="text/javascript" src="errorHandler.js"></script>
+	<script type="text/javascript" src="jquery.min.js"></script>
+	<script type="text/javascript" src="json2.min.js"></script>
+</head>
+<body>
+	<div id="loading">
+		<i class="icon-spinner icon-2x"></i>
+		<div>
+			Loading...<br><br>
+			If this takes longer than 30 seconds, it will automatically reload a maximum of 5 times.<br>
+			If it <b>still</b> doesn't work, use the bug report button at the top right of the window.
+		</div>
+	</div>
+	<div id="messages">
+
+	</div>
+	<div id="userBar" style="display: none;">
+		<div id="ping">
+			<i class="icon-circle" id="pingDot"></i>
+			<span class="ms" id="pingMs">--ms</span>
+		</div>
+		<div id="options">
+			<a href="#" class="toggle" id="toggleOptions" title="Options"><i class="icon-cog"></i></a>
+			<div class="sub" id="subOptions">
+				<a href="#" class="decreaseFont" id="decreaseFont"><span>Decrease font size</span> <i class="icon-font">-</i></a>
+				<a href="#" class="increaseFont" id="increaseFont"><span>Increase font size</span> <i class="icon-font">+</i></a>
+				<a href="#" class="togglePing" id="togglePing"><span>Toggle ping display</span> <i class="icon-circle"></i></a>
+				<a href="#" class="highlightTerm" id="highlightTerm"><span>Highlight string</span> <i class="icon-tag"></i></a>
+				<a href="#" class="saveLog" id="saveLog"><span>Save chat log</span> <i class="icon-save"></i></a>
+				<a href="#" class="clearMessages" id="clearMessages"><span>Clear all messages</span> <i class="icon-eraser"></i></a>
```

Comment:
Literally why did you remove the custom font option, I demand a reason and why I shouldn't re-add it.

We don't differentiate between languages with fonts anymore, either. We use icons for it.